### PR TITLE
Add next and previous posts to liquid context

### LINKED ISSUE #139: Get next and previous posts
This is an important feature to have for any blog. See [this blog post by David Elbe](http://david.elbe.me/jekyll/2015/06/20/how-to-link-to-next-and-previous-post-with-jekyll.html) for its usage in Jekyll.

### PR DESCRIPTION
**Pull request by nott**
Fixes #139

The idea how to use this feature in Jekyll has been described in [this blog post by David Elbe](http://david.elbe.me/jekyll/2015/06/20/how-to-link-to-next-and-previous-post-with-jekyll.html).

Same in cobalt:
```
<div>
  {% if previous.path %}
    <a class="prev" href="{{previous.path}}">&laquo; {{previous.title}}</a>
  {% endif %}
  {% if next.path %}
    <a class="next" href="{{next.path}}">{{next.title}} &raquo;</a>
  {% endif %}
</div>
```

### CODE REVIEW

File: src/document.rs [hunk 1]
```diff
@@ -303,24 +303,29 @@ impl Document {
...
-    ///
-    /// * content is inserted to the attributes of the document
-    /// * content is inserted to context
-    /// * layout may be inserted to layouts cache
+    /// Renders content and adds it to the attributes of the document and to the context.
     ///
     /// When we say "content" we mean only this document without extended layout.
+    pub fn render_content(&mut self, context: &mut Context, source: &Path) -> Result<()> {
+        let content_html = try!(self.render_html(&self.content, context, source));
+        self.attributes.insert("content".to_owned(), Value::Str(content_html.clone()));
```
Comment (nott) [inline src/document.rs]: i probably need to make "content" a static constant

**Comment by johannhof** [comment]
Sorry for the delay!

I don't really understand why the changes apart from cobalt.rs line 137-149 are necessary :)

Can you clarify what this does?

**Comment by nott** [comment]
@johannhof i will add some examples and screenshots later today or tomorrow.

**Comment by nott** [comment]
oh
haven't found time to update this
but it's still on my radar

**Comment by johannhof** [comment]
Cool, let me know when you have an update :)

**Comment by epage** [comment]
Drat, sorry, I didn't notice #218 is the same fix.  Is there a reason to keep this PR open with that merged?

**Comment by nott** [comment]
Closing this.

**Pull request closed without merging**